src/components/VaultAdjustmentDialog.tsx

```diff
@@ -25,10 +25,12 @@ const VaultAdjustmentDialog = ({ isOpen, onClose }: Props) => {
     setViewMode(ViewMode.Manage);
   };
 
+  const body = <p>{message}</p>;
+
   return (
     <BaseDialog
       title="Success: Offer Submitted"
-      message={message}
+      body={body}
```

Comment:
consider inlining, unless that doesn't work for some reasoN?
```suggestion
      body={<p>{message}</p>}
```

Reply:
Nope that works I just wasn't sure if that was too messy. Done.